WordPressKit/WordPressComRESTAPIInterfacing.h

```diff
@@ -4,13 +4,17 @@
 
 @property (strong, nonatomic, readonly) NSURL * _Nonnull baseURL;
 
+/// - Note: `parameters` has `id` instead of the more common `NSObject *` as its value type so it will convert to `AnyObject` in Swift.
+///         In Swift, it's simpler to work with `AnyObject` than with `NSObject`. For example `"abc" as AnyObject` over `"abc" as NSObject`.
 - (NSProgress * _Nullable)get:(NSString * _Nonnull)URLString
-                   parameters:(NSDictionary<NSString *, NSObject *> * _Nullable)parameters
+                   parameters:(NSDictionary<NSString *, id> * _Nullable)parameters
```

Comment:
Does the explanation make sense?

Reply:
Yep. It makes sense to me. Some Foundation API uses the same "fake generic" signature, i.e. [UserDefaults](https://developer.apple.com/documentation/foundation/nsuserdefaults/1408563-dictionaryforkey)

Reply:
Awesome! TIL